### PR TITLE
Add ability to compare TaxedMoneyField with django Fields

### LINKED ISSUE #61: TaxedMoneyField should be Django Field type
Right now there is:
```
class TaxedMoneyField(object):
...
```
is being used and named as a Field, so I guess it should also be type of some Django Field, not `object`.

### PR DESCRIPTION
**Pull request by akjanik**
Closes #61.

### CODE REVIEW

File: django_prices/models.py [hunk 1]
```diff
@@ -114,6 +118,16 @@ def __set__(self, instance, value):
         setattr(instance, self.net_field, net)
         setattr(instance, self.gross_field, gross)
 
+    def __eq__(self, other):
+        if isinstance(other, Field):
```
Comment (patrys) [inline django_prices/models.py]: I think we should also handle the case where `other` is another instance of `TaxedMoneyField`. Same applies to `__lt__`.

File: django_prices/models.py [hunk 2]
```diff
@@ -119,12 +119,12 @@ def __set__(self, instance, value):
         setattr(instance, self.gross_field, gross)
 
     def __eq__(self, other):
-        if isinstance(other, Field):
+        if isinstance(other, Field) or isinstance(other, TaxedMoneyField):
```
Comment (patrys) [inline django_prices/models.py]: You can pass a tuple: `isinstance(other, (Field, TaxedMoneyField))`

File: tests/test_prices.py [hunk 3]
```diff
@@ -8,17 +8,15 @@
...
-
 from django_prices import forms, widgets
 from django_prices.models import MoneyField, TaxedMoneyField
-from django_prices.templatetags import prices
-from django_prices.templatetags import prices_i18n
-from django_prices.validators import (
-    MaxMoneyValidator, MinMoneyValidator, MoneyPrecisionValidator)
+from django_prices.templatetags import prices, prices_i18n
+from django_prices.validators import (MaxMoneyValidator, MinMoneyValidator,
+                                      MoneyPrecisionValidator)
```
Comment (patrys) [inline tests/test_prices.py]: Please use `isort` with mode set to `4`

**Comment by akjanik** [comment]
@rafalp I found that adding or removing "field" that is `TaxedMoneyField` type is not being detected by `makemigrations` - I guess because it is not django's `Field` type.

**Comment by patrys** [comment]
@akjanik This is expected. We don't want it to be detected by migrations since there is underlying database field.

**Pull request merged**